### PR TITLE
Contribution: Fixed the quick links in the footer (not proper mapping) && fixed readme.md (command for running local dev enviornment)

### PR DESCRIPTION
**Pull request by medakshchoudhary**
I fixed the quick links in the footer of the main page as it was not properly mapped and neither was it redirected to the proper page that is /playground and others.

I also fixed the command for running the dev environment as mentioned in the default readme.md

My first contribution as a developer in the open source world.
Thanks for this oppurtunity.

### CODE REVIEW

**Comment by MajXin** [comment]
I see the commit is working well. Good job Daksh. Also congrats on your first commit

**Comment by medakshchoudhary** [comment]
> I see the commit is working well. Good job Daksh. Also congrats on your first commit

thank you saksham for the support

**Comment by medakshchoudhary** [comment]
image
image

**Comment by arjunatapadkar** [comment]
Only give link for  the available pages

**Comment by medakshchoudhary** [comment]
> Only give link for the available pages

I didn't understood what are you trying to say?

File: src/pages/Homepage.jsx
Comment (arjunatapadkar) [inline src/pages/Homepage.jsx:1]: There are no such pages available right now 
/challenges
/Community
Etc

Comment (medakshchoudhary) [reply src/pages/Homepage.jsx:1]: i fixed the quick links redirection in the footer the playground wasn't working in the quick links

Comment (arjunatapadkar) [reply src/pages/Homepage.jsx:1]: Share the snapshots, video of what exactly you changed and will merge this

Comment (medakshchoudhary) [reply src/pages/Homepage.jsx:1]: Sure lemme reach home

Comment (medakshchoudhary) [reply src/pages/Homepage.jsx:1]: https://github.com/user-attachments/assets/427929ec-a2e7-47ba-b9f2-45ccd7d6e339

I am sick so please increase the volume or use headphones or rather VLC and increase the volume sorry for the inconivence.I have described all the changes that I made 
thanks

Comment (medakshchoudhary) [reply src/pages/Homepage.jsx:1]: please can you revert back?

Comment (arjunatapadkar) [reply src/pages/Homepage.jsx:1]: Inside navs array just change the page links to ''/upcoming "  for remaining pages other than playground

Comment (medakshchoudhary) [reply src/pages/Homepage.jsx:1]: okay

Comment (medakshchoudhary) [reply src/pages/Homepage.jsx:1]: done you can check

**Review by arjunatapadkar — COMMENTED** [review]
Is it works fine now ??

**Comment by arjunatapadkar** [comment]
Checkout issue 106 @medakshchoudhary

**Comment by medakshchoudhary** [comment]
> Checkout issue 106 @medakshchoudhary

yes it is now working fine now 

also I am a little busy ue to college I will definitely check the issue you are referring to at the earliest 

thanks for the merge

**Pull request merged**